frontend/app-development/features/appContentLibrary/AppContentLibrary.test.tsx

```
@@ -7,69 +7,48 @@ import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+  ...jest.requireActual('@studio/content-library'),
+  ResourceContentLibraryImpl: mockContentLibrary,
+}));
+
+function mockContentLibrary(
+  ...args: ConstructorParameters<typeof ResourceContentLibraryImpl>
+): Partial<ResourceContentLibraryImpl> {
+  constructor(...args);
+  return { getContentResourceLibrary };
+}
+
+const constructor = jest.fn();
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)